src/type/validate.js

```diff
@@ -355,6 +358,21 @@ function validateObjectInterfaces(
   });
 }
 
+function validateInterfaces(
+  context: SchemaValidationContext,
+  iface: GraphQLInterfaceType,
+): void {
+  const possibleTypes = context.schema.getPossibleTypes(iface);
+
+  if (possibleTypes.length === 0) {
+    context.reportError(
+      `No concrete types found for Interface type ${iface.name}. ` +
+        `If only referenced via abstraction, add concrete types to schema.types array`,
```

Comment:
The schema can be create using SDL syntax so `add concrete types to schema.types array` part would be confusing in such case.
`concrete types` - better to avoid inventing new terminology.

Something like this would be great:
`Interface ${iface.name} should be implemented by at least one object type.`
Disclaimer: I'm grammatically challenged 😄

Reply:
good call, fixed!